diff --git a/kallithea/config/application.py b/kallithea/config/application.py
index b0330b5..17a178b 100644
--- a/kallithea/config/application.py
+++ b/kallithea/config/application.py
@@ -35,7 +35,7 @@ def wrap_app(app):
     app = SimpleGit(app, config)
 
     # Enable https redirects based on HTTP_X_URL_SCHEME set by proxy
-    if any(asbool(config.get(x)) for x in ['url_scheme_variable', 'force_https', 'use_htsts']):
+    if config.get('url_scheme_variable') or asbool(config.get('force_https')) or asbool(config.get('use_htsts')):
         app = HttpsFixup(app, config)
 
     app = PermanentRepoUrl(app, config)
